contents/docs/session-replay/privacy.mdx

```diff
@@ -94,4 +99,67 @@ To do so, you should add the CSS class name `ph-no-capture` to elements which sh
 <div class="ph-no-capture">I won't be captured at all!</div>
 ```
 
-Note that adding `ph-no-capture` will also prevent any [autocapture](/docs/data/autocapture) events from being captured from that element.
\ No newline at end of file
+Note that adding `ph-no-capture` will also prevent any [autocapture](/docs/data/autocapture) events from being captured from that element.
+
+
+# Common example configs
+
+## Maximum privacy - mask everything
+
+```
+{
+    maskAllInputs: true,
+    maskTextSelector: "*"
+}
+```
+
+## Good privacy - try to mask all email / password related things
```

Comment:
```suggestion
## Masking email and password
```

Reply:
I'd really like to keep the original wording. In particular because the example does not actually guarantee emails will be masked, it is just best effort